provider/openstack/networking.go

```diff
@@ -241,10 +251,6 @@ func getExternalNeutronNetworksByAZ(e *Environ, azName string) ([]string, error)
 	}
 	netIds := make([]string, 0)
 	for _, network := range networks {
-		// TODO (hml): OpenStack Compute and Network AZs have no direct relation,
```

Comment:
has something changed here?

Reply:
Drive by: should have been removed with PR 7905, which resolved this problem.